protected/humhub/modules/space/views/membership/requestMembership.php

```diff
@@ -39,6 +39,16 @@
         </div>
         <div class="modal-footer">
             <hr/>
+            <?= Html::button(
```

Comment:
Why should we still use `Html::button()` here when you could use `ModalButton::cancel()`?